builder-support/dockerfiles/Dockerfile.target.centos-8

```diff
@@ -12,7 +12,13 @@ FROM amd64/centos:8 as dist-base
 @IF [ ${BUILDER_TARGET} = centos-8-arm64 ]
 FROM arm64v8/centos:8 as dist-base
 @ENDIF
+
 ARG BUILDER_CACHE_BUSTER=
+
+@IF [[ $BUILDER_TARGET = *stream* ]]
```

Comment:
Above you use "centos-8-stream"

Reply:
err, missed the asterisks... but I do think `centos-*-stream` is better

Reply:
done!